network/commons/subnets.go

```diff
@@ -0,0 +1,185 @@
+package commons
+
+import (
+	"encoding/hex"
+	"fmt"
+	"github.com/prysmaticlabs/go-bitfield"
+	spectypes "github.com/ssvlabs/ssv-spec/types"
+	"math"
+	"math/big"
+	"strconv"
+	"strings"
+)
+
+const (
+	// SubnetsCount returns the subnet count for this fork
+	SubnetsCount = 128
+
+	UnknownSubnetId = math.MaxUint64
+
+	// UnknownSubnet is used when a validator public key is invalid
+	UnknownSubnet = "unknown"
+
+	topicPrefix = "ssv.v2"
+)
+
+// BigIntSubnetsCount is the big.Int representation of SubnetsCount
+var bigIntSubnetsCount *big.Int
+
+func init() {
+	bigIntSubnetsCount = new(big.Int).SetUint64(SubnetsCount)
+}
+
+// SubnetTopicID returns the topic to use for the given subnet
+func SubnetTopicID(subnet uint64) string {
+	if subnet == UnknownSubnetId {
+		return UnknownSubnet
+	}
+	return fmt.Sprintf("%d", subnet)
+}
+
+func CommitteeTopicID(cid spectypes.CommitteeID) []string {
+	return []string{fmt.Sprintf("%d", CommitteeSubnet(cid))}
+}
+
+// GetTopicFullName returns the topic full name, including prefix
+func GetTopicFullName(baseName string) string {
+	return fmt.Sprintf("%s.%s", topicPrefix, baseName)
+}
+
+// GetTopicBaseName return the base topic name of the topic, w/o ssv prefix
+func GetTopicBaseName(topicName string) string {
+	return strings.Replace(topicName, fmt.Sprintf("%s.", topicPrefix), "", 1)
```

Comment:
```suggestion
	return strings.TrimPrefix(topicName, topicPrefix+".")
```
Do you think this might be more readable or do you prefer your version?

Reply:
I think it's maybe Yosher's sketch that I took "as is" ;) 

I prefer the one with `TrimPrefix`, applied